arch/arm64/src/zynq-mpsoc/Kconfig

```diff
@@ -35,6 +35,136 @@ config ZYNQ_MPSOC_UART1
 	---help---
 		Select to enable support for UART1.
 
+config ZYNQ_ENET
+	bool "Ethernet"
+	default n
+	select ARCH_HAVE_PHY
+	select ARCH_HAVE_NETDEV_STATISTICS
+
 endmenu # XILINX ZYNQ_MPSOC Peripheral Selection
 
+menu "Ethernet Configuration"
+	depends on ZYNQ_ENET
+
+choice
+	prompt "ZYNQ Ethernet Interface"
+	default ZYNQ_ENET4
+	depends on ZYNQ_ENET
+
+config ZYNQ_ENET1
+	bool "ENET 1"
+
+config ZYNQ_ENET2
+	bool "ENET 2"
+
+config ZYNQ_ENET3
+	bool "ENET 3"
+
+config ZYNQ_ENET4
+	bool "ENET 4"
+
+endchoice # ZYNQ Ethernet Interface
+
+config ZYNQ_GMAC_NRXBUFFERS
+	int "Number of RX buffers"
+	default 16
+	---help---
+		GMAC buffer memory is segmented into 128 byte units (not
+		configurable).  This setting provides the number of such 128 byte
+		units used for reception.  This is also equal to the number of RX
+		descriptors that will be allocated  The selected value must be an
+		even power of 2.
+
+config ZYNQ_GMAC_NTXBUFFERS
+	int "Number of TX buffers"
+	default 8
+	---help---
+		GMAC buffer memory is segmented into full Ethernet packets (size
+		NET_BUFSIZE bytes).  This setting provides the number of such packets
+		that can be in flight.  This is also equal to the number of TX
+		descriptors that will be allocated.
+
+config ZYNQ_GMAC_PREALLOCATE
+	bool "Preallocate buffers"
+	default n
+	---help---
+		Buffer an descriptor many may either be allocated from the memory
+		pool or pre-allocated to lie in .bss.  This options selected pre-
+		allocated buffer memory.
+
+config ZYNQ_GMAC_NBC
+	bool "Disable Broadcast"
+	default n
+	---help---
+		Select to disable receipt of broadcast packets.
```

Comment:
Does this disable receipt or transmit of broadcast packets? I think the title should make it clear, so instead of "Disable Broadcast" maybe "Disable Receiving Broadcasts"?

Reply:
done! thank you very much